pkg/core/resources/apis/mesh/circuit_breaker_validator_test.go

```diff
@@ -128,6 +134,57 @@ var _ = Describe("CircuitBreaker", func() {
                  message: has to be in [0.0 - 100.0] range
                - field: conf.detectors.failure.threshold
                  message: has to be in [0.0 - 100.0] range`}),
+			Entry("empty thresholds section", testCase{
+				circuitBreaker: `
+                sources:
+                - match:
+                    kuma.io/service: frontend
+                    region: us
+                destinations:
+                - match:
+                    kuma.io/service: backend
+                conf:
+                  detectors:
+                    totalErrors: {}
+                    standardDeviation: {}
+                  thresholds: {}`,
+				expected: `
+               violations:
+               - field: conf.thresholds
+                 message: can't be empty`}),
+			Entry("empty detectors section", testCase{
+				circuitBreaker: `
+                sources:
+                - match:
+                    kuma.io/service: frontend
+                    region: us
+                destinations:
+                - match:
+                    kuma.io/service: backend
+                conf:
+                  detectors: {}
+                  thresholds:
+                    maxConnections: 2`,
+				expected: `
+               violations:
+               - field: conf.detectors
+                 message: can't be empty`}),
+			Entry("empty detectors and thresholds section", testCase{
+				circuitBreaker: `
+                sources:
+                - match:
+                    kuma.io/service: frontend
+                    region: us
+                destinations:
+                - match:
+                    kuma.io/service: backend
+                conf:
+                  detectors: {}
+                  thresholds: {}`,
+				expected: `
+               violations:
+               - field: conf
+                 message: must have at least one of the detector or threshold configured`}),
 		)
```

Comment:
add test when there is no`tresholds`

Reply:
test `one detector with default values` already checks it

Reply:
👍